bin/openssh-ldap-publickey

```diff
@@ -120,16 +131,16 @@ sub parse_nss_base_passwd {
     my $nss_base_passwd = shift;
 
     my $result = {
-        'nss_base_passwd' => undef,
-        'scope'           => undef,
+        'base_passwd' => undef,
+        'scope'       => undef,
     };
 
     # Support nss_base_passwd with scope notation
     # example:
     # nss_base_passwd ou=People?one
     if ($nss_base_passwd =~ /([^\?]*)(\?([^\?]*))?/) {
         if (defined($1) && length($1)) {
-            $result->{'nss_base_passwd'} = $1;
```

Comment:
does it brake the old clients?

Reply:
I don't see how it would, this is just a rename of an internal variable everywhere. I did it because [nslcd.conf](https://linux.die.net/man/5/nslcd.conf) has a `base passwd ...` syntax which is really the same thing as `nss_base_passwd` elsewhere, and it makes sense to me to use the same field for them both.